lib/livebook/hubs/personal.ex

```diff
@@ -131,6 +134,56 @@ defmodule Livebook.Hubs.Personal do
   def generate_secret_key() do
     Base.url_encode64(:crypto.strong_rand_bytes(@secret_key_size), padding: false)
   end
+
+  @doc """
+  Get the file systems list from storage.
+  """
+  @spec get_file_systems() :: list(FileSystem.t())
+  def get_file_systems() do
+    Storage.all(@file_systems_namespace)
+    |> Enum.sort_by(&Map.get(&1, :order, System.os_time()))
```

Comment:
```suggestion
    |> Enum.sort_by(&Map.get(&1, :order, 0))
```

Although I wonder if we should always just sort alphabetically based on the ID or bucket name or something.

Reply:
Actually, do we need to sort it here and inside `Hub`?

Reply:
I don't think so, tbh